### ISSUE TITLE
Player name garbled

### ISSUE THREAD

**Issue by BaLaLaLs** [vytrol/ProExt#7]
When the player name is not in English, garbled characters may appear, I modified the imgui initialization and added other fonts to fix this issue
```
    let mut file = File::open("C:/Windows/Fonts/msyh.ttc").expect("Failed open font file");
    let mut buffer = Vec::new();
    file.read_to_end(&mut buffer).expect("Failed read font file");
    imgui_context.fonts().add_font(&[
        FontSource::DefaultFontData { config: None },
        FontSource::TtfData {
            data: include_bytes!("C:/Windows/Fonts/msyh.ttc"),
            size_pixels: 13.0,
            config: Some(FontConfig {
                oversample_h: 4,
                oversample_v: 4,
                glyph_ranges: FontGlyphRanges::chinese_full(),
                ..FontConfig::default()
            }),
        }
    ]);
```

I am not proficient Rust and I am not sure if this is the best solution.

**Comment by vytrol** [comment]
Thanks, I'll look into this!

**Comment by vytrol** [comment]
Resolved in 393cea2
![proext_MPALC9F5ug](https://github.com/vytrol/ProExt/assets/63375371/c58e81e6-5630-4c71-93bd-c7eb3bd8c846)
